ReactAndroid/src/main/java/com/facebook/react/modules/timepicker/TimePickerMode.java

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
I think better to inline this constants in the time picker module file

Reply:
I wanted to stay iso with the `DatePicker` implementation:
https://github.com/Kerumen/react-native/blob/4e58d510958a96a6d307d7b0ba3bd8b47f70c563/ReactAndroid/src/main/java/com/facebook/react/modules/datepicker/DatePickerMode.java